src/lib/web-component/filter-timestamp/filter-timestamp-element.js

```diff
@@ -0,0 +1,86 @@
+export class FilterTimestamp extends HTMLElement {
+  constructor() {
+    super();
+
+    this.timeStamp = {};
+    this.onHiddenListener = () => browser.menus.removeAll();
```

Comment:
this onHiddenListener doesn't seem to be using this anywhere, could just be a method of the FilterTimestamp class instead of an arrow function set on a class instance parameter?

Reply:
done :-)